### ISSUE TITLE
[Documentation] Readme.md

### ISSUE THREAD

**Issue by LaurentDardenne** [PoshCode/ModuleBuilder#38]
The help content of the Build-Module function says :

>  _DESCRIPTION
>   2. Source subfolders in the same directory as the manifest:
>   Enum, Classes, Private, Public contain ps1 files_

and the readme.md says :

>  _By convention, use folders named "Classes", "Private", and "Public"_

Should this document be updated ?

**Comment by Jaykul** [comment]
*sigh* 😑 

I don't want to change it.

I wrote the readme without input, but when I wrote the function (and the help), I had a whole team of people looking over my shoulder who wanted additional folders in the list. I managed to talk them out of all but that one. 😉 Ultimately, having an extra folder or two in the defaults is not a big deal, in any case.

However, as a developer, I don't think enums should be separate from classes, and I don't want to recommend doing that. I feel like if I do, people will ask why, and I'll have to explain, like I am now... 🤔 

Bottom line: the help documents the default values, the ReadMe documents which ones I recommend you actually use. Obviously, you don't even need all of those...

**Comment by LaurentDardenne** [comment]
Whether the 'Enum' directory exists or not is a choice that I respect, as it suits me.
My question was about the presence of different information, in this case we always wonder what is the right information :-)

> Obviously, you don't even need all of those...

I agree, but here I must read the documentation and the source code.
If I offer this tool to my colleagues at work, I must point this out.
